tests/test_trees.py

```diff
@@ -233,6 +233,24 @@ def b(cls, args):
         x = MyTransformer().transform( t )
         self.assertEqual(x, t2)
 
+    def test_inline(self):
+        class MyTransformer(Transformer):
+            def b(self, children):
+                return 'b'
+
+            def a(self, children):
+                return Inline('c', 'd')
+
```

Comment:
Adding a test to test for a Token would be nice just in case we somehow break that in the future.